src/schemas.py

```
@@ -253,23 +261,11 @@ class DialecticOptions(BaseModel):
         None,
         description="Optional peer to get the representation for, from the perspective of this peer",
     )
-    queries: str | list[str]
+    query: Annotated[
+        str, Field(min_length=1, max_length=10000, description="Dialectic API Prompt")
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)